src/runtime_src/core/tools/xbmgmt2/ReportPlatform.cpp

```diff
@@ -301,22 +302,26 @@ ReportPlatform::writeReport( const xrt_core::device* /*_pDevice*/,
                              const std::vector<std::string>& /*_elementsFilter*/,
                              std::ostream & _output) const
 {
+  auto pretty_print =[](std::string str) {

```

Comment:
Please change the name.....too generic.  Especially since `pretty_print` usually implies not changing the data being returned only the format.

Some suggestions:
```
string_or_NA
if_empty_NA
```